src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,598 @@
+
+`shadowMode: none` disables shadowing.
+
+- The LMS sends application requests to the source of truth only.
+- Results from the source of truth are returned to the application.
+- Writes must be manually replicated from the source database to the target database.
+
+You can use this mode to perform a [consistent cutover](#consistent-cutover), along with another tool such as [change data capture (CDC)]({% link {{ page.version.version }}/cdc-queries.md %}) that replicates writes to the target database. {% comment %}For an example, see [Consistent cutover without shadowing](#consistent-cutover-without-shadowing).{% endcomment %}
+
+### `async`
+
+<img src="{{ 'images/v23.2/migrations/lms_async.svg' | relative_url }}" alt="MOLT LMS shadowing mode - async" />
+
+`shadowMode: async` writes to both databases.
+
+- The LMS sends application requests to the source of truth and target database in asynchronous threads, and waits only for the source of truth to respond.
+- Results from the source of truth are returned to the application.
+
+You can use this mode to confirm that your queries succeed on CockroachDB without verifying performance or correctness.
+
+### `sync`
+
+<img src="{{ 'images/v23.2/migrations/lms_sync.svg' | relative_url }}" alt="MOLT LMS shadowing mode - sync" />
+
+`shadowMode: sync` writes to both databases.
+
+- The LMS sends application requests to the source of truth and the target database, and waits for each to respond.
+- Results from the source of truth are returned to the application.
+- Results from the non-source of truth are discarded.
+
+You can use this mode to perform an [immediate cutover](#immediate-cutover).
+
+### `strict-sync`
+
+<img src="{{ 'images/v23.2/migrations/lms_strict-sync.svg' | relative_url }}" alt="MOLT LMS shadowing mode - strict-sync" />
+
+`shadowMode: strict-sync` writes to both databases and enforces correctness on both databases.
+
+- The LMS sends application requests to the source of truth and the target database, and waits for each to respond.
+- Results from the source of truth are returned to the application.
+- If the non-source of truth returns an error, that error is returned instead of the result from the source of truth.
+
+You can use this mode to perform an [immediate cutover](#immediate-cutover).
+
+## Perform a cutover
+
+### Consistent cutover
+
+A consistent cutover maintains data consistency with [minimal downtime]({% link {{ page.version.version }}/migration-overview.md %}#minimal-downtime). When using the LMS, consistent cutover is handled using the [`molt-lms-cli`](#molt-lms-cli) commands `cutover consistent begin` and `cutover consistent commit`, between which downtime occurs. The amount of downtime depends on the maximum duration of any transactions and queries that need to complete, and the time it takes for replication to catch up from the source to the target database.
```

Comment:
```suggestion
A consistent cutover maintains data consistency with [minimal downtime]({% link {{ page.version.version }}/migration-overview.md %}#minimal-downtime). When using the LMS, consistent cutover is handled using the [`molt-lms-cli`](#molt-lms-cli) commands `cutover consistent begin` and `cutover consistent commit`, during which application requests are queued and will be responded to after cutover. This delay in response time is related to the maximum duration of any transactions and queries that need to complete, and the time it takes for replication to catch up from the source to the target database. This is typically 1-2 minutes in a production system.
```
I want to get really specific in this section about what downtime means. It's not really an "outage" but rather a delay in responses. Everything still gets processed as long as requests don't time out so we want to be specific.

Reply:
Noted and updated!